prow/plugins/label/label.go

```diff
@@ -226,19 +230,29 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	}
 
 	toRepeat := []string{}
+	isMember := false
 	if len(sigMatches) > 0 {
-		isMember, _ := gc.IsMember(ae.org, ae.login)
-		if !isMember {
-			toRepeat = ae.getRepeats(sigMatches, existingLabels)
-		}
+		isMember, _ = gc.IsMember(ae.org, ae.login)
+		toRepeat = ae.getRepeats(sigMatches, existingLabels)
 	}
 
 	if len(toRepeat) > 0 {
-		msg := fmt.Sprintf(chatBack, strings.Join(toRepeat, ", "))
-		if err := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {
-			log.WithError(err).Errorf("Could not create comment \"%s\".", msg)
+		if !isMember {
+			msg := fmt.Sprintf(chatBack, strings.Join(toRepeat, ", "))
+			if err := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {
+				log.WithError(err).Errorf("Could not create comment \"%s\".", msg)
+			}
 		}
 
+		if sc != nil {
+			//if sig matches then send a notification on slack
+			for _, sig := range sigMatches {
+				msg := fmt.Sprintf("%s\nIssue:%d, %s\nUrl:%s", ae.body, ae.issue.Number, ae.issue.Title, ae.issue.HTMLURL)
```

Comment:
Slack supports [block quoting](https://get.slack.help/hc/en-us/articles/202288908-Format-your-messages#blockquotes). You could use that on the comment body.